spec/lib/tmuxinator/window_spec.rb

```diff
@@ -208,23 +229,54 @@
     end
   end
 
-  describe "#tmux_new_window_command" do
-    let(:project) { double(:project) }
-    let(:window) { Tmuxinator::Window.new(yaml, 0, project) }
-    let(:root?) { true }
-    let(:root) { "/project/tmuxinator" }
+  describe "#synchronize" do
+    context "synchronization enabled" do
+      let(:synchronize) { true }
 
-    before do
-      allow(project).to receive_messages(
-        name: "test",
-        tmux: "tmux",
-        root: root,
-        root?: root?,
-        base_index: 1
-      )
+      it "should have synchronization enabled" do
+        expect(window.synchronize).to be true
+      end
     end
 
-    let(:tmux_part) { project.tmux }
+    context "synchronization disabled" do
+      let(:synchronize) { false }
+
+      it "should have synchronization enabled" do
```

Comment:
minor: 
I believe this should read `it "should have synchronization disabled" do`